lib/models/builder.js

```diff
@@ -226,6 +253,58 @@ class Builder extends CoreObject {
   finalizeBuild() {
     this.project.configCache.clear();
   }
+
+  /**
+   * broccoli-builder reformats the response into {directory, graph}, this method is a backwards
+   * compatible shim for broccoli 1.x
+   * @private
+   * @method compatNode
+   * @param node The node returned from Broccoli builder
+   */
+  compatNode(node) {
+    if (this.broccoli2) {
```

Comment:
I was hoping that we could structure the code such that the broccoli@2 case is the "normal" case, and we do any shimming/polyfilling for the older broccoli-builder APIs...

Reply:
I think we should do that once we move out of experimental mode